openedx/core/djangoapps/content/learning_sequences/models.py

```diff
@@ -113,6 +113,10 @@ class LearningSequence(TimeStampedModel):
     )
     usage_key = UsageKeyField(max_length=255)
 
+    # A URL-safe Base64-encoding of a blake2b hash of the usage key.
+    # For aesthetic use (eg, as a path parameter, to shorten URLs).
+    usage_key_hash = models.CharField(max_length=255, blank=True, null=True)
```

Comment:
It can be a lot smaller than this, right?

Reply:
Yes, good point.

Reply:
I changed the `max_length` to `USAGE_KEY_HASH_LENGTH`, which is 8.